lib/pages/chat/message_edit_field.dart

```diff
@@ -212,85 +212,90 @@ class _MessageEditField extends State<MessageEditField> {
-                                      ),
-                                    ),
-                                    Expanded(child: SizedBox()),
-                                    Icon(
-                                      Icons.message_rounded,
-                                      color: Colors.grey,
-                                    )
-                                  ],
+                                  ),
+                                ),
+                                SizedBox(
+                                  width: 15,
                                 ),
-                                onTap: () {
-                                  BlocProvider.of<MentionsCubit>(context)
-                                      .clearMentions();
-                                  mentionReplace(state.users[index].username);
-                                  setState(() {
-                                    _mentionsVisible = false;
-                                  });
-                                },
-                              ),
-                            );
-                          },
+                                Text(
+                                  '${state.users[i].firstName} ',
+                                  style: TextStyle(
+                                      fontSize: 16.0,
+                                      fontWeight: FontWeight.w300),
+                                ),
+                                Text(
+                                  ' ${state.users[i].lastName}',
+                                  style: TextStyle(
+                                    fontSize: 16.0,
+                                  ),
+                                ),
+                                Expanded(child: SizedBox()),
+                                Icon(
+                                  Icons.message_rounded,
+                                  color: Colors.grey,
+                                )
+                              ],
+                            ),
+                            onTap: () {
+                              BlocProvider.of<MentionsCubit>(context)
+                                  .clearMentions();
+                              mentionReplace(state.users[i].username);
+                              setState(() {
+                                _mentionsVisible = false;
+                              });
+                            },
+                          ),
                         );
-                      } else if (state is MentionsEmpty) {
-                        return Container();
-                        //Text("Empty");
+                        if (i < state.users.length - 1) {
+                          _listW.add(Divider(thickness: 1));
+                        }
                       }
+                      return ConstrainedBox(
+                        constraints: BoxConstraints(maxHeight: 250),
```

Comment:
Are you sure that fixing the height is a good idea? It might still overflow on small screens. But if you checked it on small screen sizes, and it's OK, please confirm it and I will merge the PR.